src/modules/topos/tps_msg.c

```diff
@@ -910,7 +910,10 @@ int tps_request_received(sip_msg_t *msg, int dialog)
 		goto error;
 	}
 	metid = get_cseq(msg)->method_id;
-	if((metid & (METHOD_BYE | METHOD_INFO | METHOD_PRACK | METHOD_UPDATE))
+	if(((metid & (METHOD_BYE | METHOD_INFO | METHOD_PRACK | METHOD_UPDATE))
+			   || ((metid & (METHOD_NOTIFY))
+					   && (msg->event
+							   && strncmp(msg->event->body.s, "talk", 4) == 0)))
```

Comment:
I would skip checking the event body to be "talk", there may be other event types and I think it is not the proxy that should decide what events to be accepted in such case.

Reply:
Alright